build/build.sh

```diff
@@ -43,3 +44,20 @@ wkhtmltopdf \
   --quiet \
   output/index.html \
   output/manuscript.pdf
+
+echo "Exporting Word Docx manuscript"
+pandoc --verbose \
+--smart \
+--from=markdown \
+--to=docx \
+--filter pandoc-fignos \
+--filter pandoc-tablenos \
+--bibliography=$BIBLIOGRAPHY_PATH \
+--csl=$CSL_PATH \
+--reference-docx=$DOCX_PATH \
+--self-contained \
+--metadata link-citations=true \
+--include-in-header=build/assets/analytics.js \
+--include-after-body=build/assets/anchors.js \
```

Comment:
I suspect both these javascript files only affect HTML output so they can be removed here.